kitsune/questions/views.py

```diff
@@ -104,8 +104,7 @@
 def product_list(request, template):
     """View to select a product to see related questions."""
     return render(request, template, {
-        'products': Product.objects.filter(
-            questions_locales__locale__in=[request.LANGUAGE_CODE])
+        'products': Product.objects.filter(questions_locales__locale=request.LANGUAGE_CODE)
```

Comment:
I have no idea why that would be using `__in` like that. That's weird.

I think this change is fine.